build_tools/circle/build_doc.sh

```diff
@@ -114,6 +114,13 @@ pip install sphinx-gallery numpydoc
 # Build and install scikit-learn in dev mode
 python setup.py develop
 
+# TO BE UNCOMMENTED BEFORE MERGE
```

Comment:
Seems strange for me. Ignore if that's intended.

Reply:
Yep good catch @qinhanmin2014, I have pushed e2584c316